app/src/main/java/com/github/sdpteam15/polyevents/helper/DatabaseHelper.kt

```diff
@@ -0,0 +1,124 @@
+package com.github.sdpteam15.polyevents.helper
+
+import com.github.sdpteam15.polyevents.model.database.remote.Database.currentDatabase
+import com.github.sdpteam15.polyevents.model.database.remote.DatabaseConstant
+import com.github.sdpteam15.polyevents.model.entity.Event
+import com.github.sdpteam15.polyevents.model.entity.Item
+import com.github.sdpteam15.polyevents.model.entity.MaterialRequest
+import com.github.sdpteam15.polyevents.model.entity.MaterialRequest.Status.*
+import com.github.sdpteam15.polyevents.model.entity.Zone
+import com.github.sdpteam15.polyevents.model.observable.Observable
+import com.github.sdpteam15.polyevents.model.observable.ObservableList
+
+/**
+ * The goal of this objects is to create function to nested relation from the database
+ */
+object DatabaseHelper {
```

Comment:
Good idea add a helper for the database

Reply:
Could be a nice idea, but we made a democraty vote on pro and cons, and you lose :)